include/zephyr/net/http/server.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2023, Emna Rekik
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef HTTP2_SERVER_H
+#define HTTP2_SERVER_H
+
+#include <zephyr/posix/poll.h>
```

Comment:
This can be switched to `<poll.h>` as long as `CONFIG_POSIX_API` is in `prj.conf`